### PR TITLE
Add DVCLive reference to Experiments "Setup Complete"

### PR DESCRIPTION
**Pull request by julieg18**
## Demo

image

Part of #3434

### CODE REVIEW

**Comment by julieg18** [comment]
> @julieg18 could you please check that the snippets also up to date with what we have in docs please

They appear to be up to date, though our code example of `huggingface` is a bit more simplified:

Docs:
```
from dvclive.huggingface import DVCLiveCallback

...

 trainer = Trainer(
    model, args,
    train_dataset=train_data,
    eval_dataset=eval_data,
    tokenizer=tokenizer,
    compute_metrics=compute_metrics,
)
trainer.add_callback(DVCLiveCallback())
trainer.train()
```

VSCode:
```
from dvclive.huggingface import DVCLiveCallback

...

trainer.add_callback(DVCLiveCallback(save_dvc_exp=True))
trainer.train()
```

I think our simplified version gets the point across though.

**Comment by julieg18** [comment]
> what happens with the page when there are no data yet?

The user will see this:

image

For the record, this pr does not alter the "No Data" section at all :)

**Comment by shcheklein** [comment]
Thanks, yes, it was more as reminder for me. I think this PR is good.

File: webview/src/setup/components/experiments/Experiments.tsx [hunk 1]
```diff
@@ -43,12 +44,17 @@ export const Experiments: React.FC<ExperimentsProps> = ({ isDvcSetup }) => {
   return (
     <EmptyState isFullScreen={false}>
       <h1>Setup Complete</h1>
+      <p>
+        Learn more about using <a href="https://dvc.org/doc/dvclive">DVCLive</a>{' '}
+        in your project in its docs.
```
Comment (mattseddon) [inline webview/src/setup/components/experiments/Experiments.tsx:49]: [C] Maybe change this to have the link on `the docs`.

Comment (mattseddon) [reply webview/src/setup/components/experiments/Experiments.tsx:49]: or `Learn more about using DVCLive in the docs` with the link on `docs`.

**Review by mattseddon — APPROVED** [review]

**Review by sroy3 — APPROVED** [review]

File: webview/src/setup/components/experiments/Experiments.tsx (hunk 1 above)
Comment (julieg18) [reply webview/src/setup/components/experiments/Experiments.tsx]: Updated!

**Pull request merged**